packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,131 @@ related: ['Button']
 import CheckIcon from '@patternfly/react-icons/dist/esm/icons/check-icon';
 import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
 
-A **modal** loads and displays important information without navigating away from the current page.
+A **modal** displays important information to a user without requiring them to navigate to a new page.
 
 ## Primary elements
-<img src="./img/modal.png" alt="Modal" />
 
-1. **Backdrop**
-2. **Modal box**
-3. **Title:** Concisely convey the purpose of the modal.
-4. **Close**
-5. **Content area**
-6. **Buttons:** Enable a user to confirm, submit, or exit. Modal dialogs can include multiple action buttons, but should be limited to one primary button
+<img src="./img/modal-call-outs.png" alt="An example of a modal in context with call outs to highlight each listed element" width="756"/>
 
-### Spacing
-#### Compact
-<img src="./img/compact.png" alt="Compact" /> 
+* **Backdrop**
+* **Modal box**
+* **Headline**
+* **Close icon**
+* **Content area**
+* **Buttons** 
 
-The compact modal dialog uses a spacing of 24px. Use this for smaller modals that don’t contain much content.
+See the "[Content](#content)" section for information about writing modal copy.
 
-#### Large
-<img src="./img/large.png" alt="Large" />
```

Comment:
Header should be more bolded! Make sure to use the modal symbol with header that's in the symbol library!